lib/iris/tests/integration/netcdf/test_general.py

```diff
@@ -487,39 +487,35 @@ def test_path_string_save_same(self):
 @tests.skip_data
 class TestWarningRepeats(tests.IrisTest):
     def test_datum_once(self):
-        fnames = [
-            "false_east_north_merc.nc",
-            "non_unit_scale_factor_merc.nc",
-            "toa_brightness_temperature.nc",
-        ]
-        fpaths = [
-            tests.get_data_path(("NetCDF", "mercator", fname)) for fname in fnames
-        ]
+        """Tests for warnings being duplicated.
 
-        with warnings.catch_warnings():
-            warnings.simplefilter("default")
-            with warnings.catch_warnings(record=True) as record:
-                for fpath in fpaths:
-                    iris.load(fpath)
-        assert len(record) == 1
+        Notes
+        -----
+        This test relies on `iris.load` throwing a warning. This warning might
+        be removed in the future, in which case `assert len(record) == 2 should`
+        be change to `assert len(record) == 1`.
 
-    def test_datum_default(self):
+        toa_brightness_temperature.nc has lazy data, and triggers a
+        specific part of dask which contains a `catch_warnings()` call,
+        so has been removed from the `fnames` list.
```

Comment:
I think it's elaborating "... a `catch_warnings()` call _which causes warnings to be repeated,_ so has been removed from the `fnames` list _until a solution is found for such a file._"